app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,18 @@
+class UserMailer < ApplicationMailer
+  default from: Rails.configuration.address_for_transactional_authentication_emails
+  helper :time
```

Comment:
I don't quite understand the `helper :time` line (probably due my Rails naievete).

Reply:
Mailers don't include helpers by default, but you can include the template helpers you want to include in this way. This will load in the TimeHelper class so that I can use those helpers to format the date properly in the view.